src/main/java/de/tum/in/www1/artemis/service/compass/controller/ModelClusterFactory.java

```diff
@@ -39,7 +38,7 @@ public class ModelClusterFactory {
      *
      * @param modelingSubmissions the submissions to build the clusters from
      * @param exercise the exercise that submissions belong to
-     * @return the clusters that have more than one element in them
+     * @return an unmodifiable list of  clusters that have more than one element in them
```

Comment:
remove one space after "of"
```suggestion
     * @return an unmodifiable list of clusters that have more than one element in them
```